src/avi/render.clj

```diff
@@ -29,7 +29,10 @@
                          (color/make :blue :black))
             line-text (or document-line "~")]
         (fill-rendition-line! rendition (+ i from-line) [line-color line-text])))
-    (fill-rendition-line! rendition to-line [(color/make :black :white) (or (:name document) "[No Name]")])))
+    (let [file-name (or (:name document) "[No Name]")
+          [i j] (:point (e/edit-context editor))
+          msg-txt (str file-name "\t[" (inc i) "," (inc j) "]" )]
```

Comment:
I think the tab character will be a problem.  The screen is drawn in such a way that all characters are expected to occupy one position and not move the cursor more than one.  A few spaces should work.  Or you could get fancy and compute the number of spaces based on the length of the filename.